jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -117,6 +121,12 @@ private static void assertDockerInspect(String imageReference)
                 + "            }"));
   }
 
+  static String assertDigestFile(Path digestPath) throws IOException, DigestException {
+    Assert.assertTrue(Files.exists(digestPath));
+    String digest = new String(Files.readAllBytes(digestPath), StandardCharsets.UTF_8);
```

Comment:
nit: `Files.readAllLines` might be easier?

Reply:
That returns a list so I think this is cleaner.